src/documents/activities/FunctionalMashup.md

```diff
@@ -6,39 +6,54 @@ eleventyNavigation:
+cards, or find other ways to create a collection of random objects.
 <figure>
-[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted on.](/assets/images/object_cards.jpg)](/assets/images/object_cards.jpg)
+[![An image showing a large selection of cards spread out on the floor, each with a different magazine image pasted
+on.](/assets/images/object_cards.jpg)](/assets/images/object_cards.jpg)
 <figcaption>
-*Figure 1:* Example of a collection of object cards created by pasting random magazine images onto sheets of paper.</figcaption>
+*Figure 1:* Example of a collection of object cards created by pasting random magazine images onto sheets of
+paper.</figcaption>
 </figure>
 
-If desired and available you can provide additional tools and materials that can be used together with the object cards to build prototypes (e.g. scissors, glue, tape, pipe cleaners, etc.).
+If desired and available you can provide additional tools and materials that can be used together with the object cards
+to build prototypes (e.g. scissors, glue, tape, pipe cleaners, etc.).
 
 ### Steps
 
 * Break into smaller groups (3 to 5 people per group is usually best)
-* Keeping the objects on the cards hidden, each group selects the same number of cards from the main deck (this number can be decided by the larger group).
-* Each group then chooses or is assigned one function from a collection of functions - these can be determined by the facilitator ahead of time or collectively by the group. The functions can be realistic or can be more experimental and exploratory. For example: communicating with a neighbour during a power failure, sending money to relatives overseas, taking your newborn baby to work, walking your dog in an airplane, looking after your parents remotely, etc.
-* The challenge for each group is to combine and use all of the objects on their cards to create something that fulfills the selected function. One way to approach this might be to tell a story about how your objects come together to fulfill the function.
+* Keeping the objects on the cards hidden, each group selects the same number of cards from the main deck (this number
+  can be decided by the larger group).
+* Each group then chooses or is assigned one function from a collection of functions - these can be determined by the
+  facilitator ahead of time or collectively by the group. The functions can be realistic or can be more experimental and
+  exploratory. For example: communicating with a neighbour during a power failure, sending money to relatives overseas,
+  taking your newborn baby to work, walking your dog in an airplane, looking after your parents remotely, etc.
+* The challenge for each group is to combine and use all of the objects on their cards to create something that fulfills
+  the selected function. One way to approach this might be to tell a story about how your objects come together to
+  fulfill the function.
```

Comment:
I think this is an example of what I meant about the bullet styling.

Reply:
@jobara I think the challenge here is that we've set our Markdown column width to 120 whereas Github wraps at 80 (I think). In my IDE this looks fine as long as the window is 120 characters wide:

<img width="919" alt="Screen Shot 2020-11-04 at 10 14 12 AM" src="https://user-images.githubusercontent.com/605361/98122103-845bed00-1e86-11eb-9316-6bf494130338.png">

That being said I'm not sure what the benefit of the line length restriction is for Markdown given that most IDEs support soft wrapping. Maybe we could discuss further.

Reply:
I actually meant that it's an issue with the rendered site. 

![Screen Shot 2020-11-04 at 12 11 00 PM](https://user-images.githubusercontent.com/556024/98143229-d7d63700-1e96-11eb-8783-b2d5dfdf1c2f.png)

Reply:
> That being said I'm not sure what the benefit of the line length restriction is for Markdown given that most IDEs support soft wrapping. Maybe we could discuss further.

@the-t-in-rtf would probably be the best to talk about that.

Reply:
I understand the point about soft wrapping and IDEs, but we should also consider PR diffs, which frankly are much more readable with at least some line length limit.

A line length of 120 characters has worked well for Infusion and other projects, especially since trailing links don't count against the length and tables aren't included.